### DISCUSSION TITLE
タグページのアイコンをクリックすると、404エラーを返す

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#206] [要望・アイデア・不具合]
[タグ一覧](https://qiita.com/tags)から任意のタグページを参照し、各タグページのアイコンに対してマウスオーバーした状態（指マーク表示時）でアイコンをクリックすると、404エラーを返す事象が発生しています。

あるべき姿としては、マウスオーバーしてもクリックできない様にするのが仕様として正しい様に思えたのですがいかがでしょうか。特に困ってはいませんが、気になったのでご報告です。添付のファイルは参考までに[pythonのタグ](https://qiita.com/tags/python/edit)をクリックしたものです。以上です。
<img width="1440" alt="スクリーンショット 2022-04-20 0 37 15" src="https://user-images.githubusercontent.com/38848514/164043355-851aea85-f5ff-4356-9ac1-5aed9611f609.png">

**Comment by kskwtnk** [comment]
@Brutus03 
ご意見ありがとうございます！

こちら、実は様々なタグのアイコンを関係ない画像に変えるいたずらが昨日発生しまして、一時対応としてスタッフ以外は変えられないように変更いたしました
アイコン変更をできる権限やフローの見直しなどを実施予定なので、今しばらくお待ちください 🙏

**Comment by wataru86** [comment]
@Brutus03
お待たせしました :bow: 
タグアイコンの変更申請ができるようになりました！
https://help.qiita.com/ja/articles/qiita-icon-edit-request

**Reply by Brutus03** [reply]
ご連絡ありがとうございます。
404エラーが出なくなったのであれば、クローズで大丈夫だと思います。
ご対応ありがとうございました。
